src/commands/dockersupport/reopenInContainer.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { AzExtTreeItem } from 'vscode-azureextensionui';
+import { localize } from '../../localize';
+import { IProjectWizardContext } from '../createNewProject/IProjectWizardContext';
+import { getProjectFolder } from './getProjectFolder';
+
+/**
+ * Explorer menu item which reopens the local app functions project in a dev container
+ */
+export async function reopenInContainer(context: IProjectWizardContext, node?: AzExtTreeItem): Promise<void> {
+
+    try {
+        if (node) {
+            const message: string = localize('selectFunctionProject', 'Select your App Functions Project');
+            const projectFilePath: string = await getProjectFolder(context, message, context.workspacePath);
+            vscode.commands.executeCommand('remote-containers.reopenInContainer', vscode.Uri.file(projectFilePath));
+        } else {
+            void vscode.window.showInformationMessage(localize('noProjectOpened', 'Conditions not met. Cannot use Docker with this project'));
+        }
+    } catch (error) {
+        const message: string = localize('failReopenInContainer', 'Failed to reopen project in a dev container.');
+        throw new Error(message);
```

Comment:
should we be throwing or showing an error message? it always scares me to see an exception being throw in client side apps.

Reply:
That's good to note, I never thought about it like that. I only need it to show an error, I will make the change.